pkg/types/types.go

```diff
@@ -68,14 +74,16 @@ type LastUpdated struct {
 	Date time.Time
 }
 type VulnerabilityDetail struct {
-	ID          string   `json:",omitempty"` // e.g. CVE-2019-8331, OSVDB-104365
-	CvssScore   float64  `json:",omitempty"`
-	CvssScoreV3 float64  `json:",omitempty"`
-	Severity    Severity `json:",omitempty"`
-	SeverityV3  Severity `json:",omitempty"`
-	References  []string `json:",omitempty"`
-	Title       string   `json:",omitempty"`
-	Description string   `json:",omitempty"`
+	ID           string   `json:",omitempty"` // e.g. CVE-2019-8331, OSVDB-104365
+	CvssScore    float64  `json:",omitempty"`
+	CvssVector   string   `json:",omitempty"`
+	CvssScoreV3  float64  `json:",omitempty"`
+	CvssVectorV3 string   `json:",omitempty"`
```

Comment:
info: We might be able to re-use `CVSSVector`
```suggestion
	CvssScore    float64  `json:",omitempty"`
	CvssScoreV3  float64  `json:",omitempty"`
	CvssVector   CVSSVector   `json:",omitempty"`
```